packages/wallets/src/components/Base/WalletText/WalletText.scss

```diff
@@ -0,0 +1,169 @@
+$color-map: (
+    general: #333333,
+    primary: #999999,
+    success: #4bb4b3,
+    warning: #ffad3a,
+    error: #ec3f3f,
+    white: #ffffff,
+    black: #000000,
+    less-prominent: #999999,
+    red: #ff444f,
+    blue: #377cfc,
+);
+
+$desktop-font-size-map: (
+    '2xs': (
+        size: 1rem,
+        line_height: 1.4rem,
+    ),
+    'xs': (
+        size: 1.2rem,
+        line_height: 1.8rem,
+    ),
+    'sm': (
+        size: 1.4rem,
+        line_height: 2rem,
+    ),
+    'md': (
+        size: 1.6rem,
+        line_height: 2.4rem,
+    ),
+    'lg': (
+        size: 2rem,
+        line_height: 3rem,
+    ),
+    'xl': (
+        size: 2.4rem,
+        line_height: 3.4rem,
+    ),
+    '2xl': (
+        size: 3.2rem,
+        line_height: 4rem,
+    ),
+    '3xl': (
+        size: 4.8rem,
+        line_height: 6rem,
+    ),
+    '4xl': (
+        size: 6.4rem,
+        line_height: 8rem,
+    ),
+    '5xl': (
+        size: 8rem,
+        line_height: 10rem,
+    ),
+);
+
+$mobile-font-size-map: (
+    '2xs': (
+        size: 0.8rem,
+        line_height: 1.2rem,
+    ),
+    'xs': (
+        size: 1rem,
+        line_height: 1.4rem,
+    ),
+    'sm': (
+        size: 1.2rem,
+        line_height: 1.8rem,
+    ),
+    'md': (
+        size: 1.4rem,
+        line_height: 2rem,
+    ),
+    'lg': (
+        mobile_fontsize: 1.6rem,
+        line_height: 2.4rem,
+    ),
+    'xl': (
+        size: 1.8rem,
+        line_height: 2.6rem,
+    ),
+    '2xl': (
+        size: 2.4rem,
+        line_height: 3rem,
+    ),
+    '3xl': (
+        size: 2.8rem,
+        line_height: 3.4rem,
+    ),
+    '4xl': (
+        size: 3.2rem,
+        line_height: 4rem,
+    ),
+    '5xl': (
+        size: 4rem,
+        line_height: 5rem,
+    ),
+);
+
+$line-height-map: (
+    '3xs': 1.2rem,
+    '2xs': 1.4rem,
+    'xs': 1.6rem,
+    'sm': 1.8rem,
+    'md': 2rem,
+    'lg': 2.2rem,
+    'xl': 2.4rem,
+    '2xl': 2.6rem,
+    '3xl': 2.8rem,
+    '4xl': 3rem,
+    '5xl': 3.2rem,
```

Comment:
Lets add 3.4rem as well. If in the future design changes, and we come and add it here. It will be a trouble.